### DISCUSSION TITLE
What really is AT protocol ? General Guide for beginner.

### DISCUSSION THREAD

**Discussion by aialok** [bluesky-social/atproto#1780] [Clarifications and "Why?"] [answered]
As a beginner and open source enthusiast, I recently came across the Bluesky GitHub. Initially, I thought Bluesky was just another Twitter-like platform, but I later discovered that AT protocol on which the Bluesky is built on and how it aims to be the next generation of social networks. However, I'm still struggling to fully understand the AT protocol.

Could someone explain what the AT protocol is and why there is a need for such a protocol? Additionally, how is Bluesky built on top of it? Unfortunately, I couldn't find any blogs or resources suitable for beginners to get started with.

Thank you !

**Comment by pfrazee** [comment, marked as answer]
Have you taken a look at https://atproto.com ?

**Reply by aialok** [reply]
@pfrazee sir, 
Can I get access to bluesky. I really interested in getting know about the bluesky and atprotocol and contribute to it.
How bluesky is build using atprotocol and what are the new and cool feature in the bluesky. 
I want to try and contribute to bluesky.
It would be very nice If I get access to the bluesky.

Thank you
Happy Coding !

**Reply by aialok** [reply]
> Have you taken a look at https://atproto.com ?

I tried checking out the atproto docs and your blogs and now I got to know lil about the atprotocol.

Thank you
